Cognite.Simulator.Utils/Configuration.cs

```diff
@@ -75,8 +75,8 @@ public override void GenerateDefaults()
                 Cognite = new CogniteConfig();
             }
 
-            if (Cognite.CdfRetries == null) {
-                Cognite.CdfRetries = new RetryConfig();
+            // the default max retries of 5 does not work well with simint
+            if (Cognite.CdfRetries.MaxRetries == 5) {
```

Comment:
any better way of doing this? looks very sketchy
we could have our own version of BaseConfig to override this on a lower level

Reply:
yeah, i dont like this approach either, let me remove this whole part and do it in a separate pr